usr/share/rear/lib/serial-functions.sh

```diff
@@ -0,0 +1,42 @@
+# Get available serial devices
+function get_serial_devices {
+    if [ -z $SERIAL_CONSOLE_DEVICES ]; then
+        serial_devices=$(ls /dev/ttyS[0-9]* /dev/hvsi[0-9]* | sort)
+        echo "$serial_devices"
+    else
+        $SERIAL_CONSOLE_DEVICES
+    fi
+}
+
+function build_cmdline {
+    # Enable serial console, unless explicitly disabled
```

Comment:
Too generic name "build_cmdline".
Better use a more specific name that tells what that function actually does
e.g. "cmdline_add_console"